src/applications/financial-status-report/components/Typeahead.jsx

```diff
@@ -92,7 +92,7 @@ const Typeahead = ({ uiSchema, idSchema, formData, onChange, onBlur }) => {
       const fetchedsuggestions = getSuggestions(input);
       setSuggestions(fetchedsuggestions);
     },
-    [], // eslint-disable-line react-hooks/exhaustive-deps
+    [getInput, getSuggestions], // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
getInput, getSuggestions added to the dependency array to update typeahead ... eslint disabled because its missing the one dependency for input but doesn't need to be included here

Reply:
Does this still need to be disabled?

Reply:
it does.. theres some ongoing work thats preventing me from refactoring but its missing a dependency that will cause it to break if added so needs to be disabled for now